cmd/main.go

```diff
@@ -0,0 +1,70 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"net/url"
+	"os"
+	"path"
+
+	"github.com/onflow/flixkit-go"
+)
+
+func main() {
+	action := flag.String("action", "", "Specify the action to take.")
+	flag.Parse()
+	
+	switch *action {
+	case "generate":
+		fmt.Println("Generating ...")
+
+		templates := []string{
+			"./templates/flow-transfer-tokens.template.json",
+			"./templates/multiply.template.json",
+			"https://flix.flow.com/v1/templates?name=transfer-flow",
+		}
+
+		for _, value := range templates {
+			GenerateTemplateBinding(value)
+		}
+	
+	default:
+		fmt.Println("Invalid or missing action.")
+	}
+}
+
+func GenerateTemplateBinding(flixQuery string) {
+	flixService := flixkit.NewFlixService(&flixkit.Config{})
+	ctx := context.Background()
+
+	code, err := flixService.GenFlixBinding(ctx, flixQuery, "javascript")
+	if err != nil {
+		fmt.Errorf("could not process flix with path of %s: %w", flixQuery, err)
+	}
+	fmt.Println(code)
+	parsedURL, err := url.Parse(flixQuery)
+	filename := path.Base(parsedURL.Path)
+	outputDirectory := path.Dir(parsedURL.Path)
+	filenameNoExt := filename[0 : len(filename)-len("json")] + "js"
+	SaveBindingFile(outputDirectory, filenameNoExt, code)
+}
+
+func SaveBindingFile(dirPath string, filename string, code string) error {
```

Comment:
This should happen in CLI after it's returned

Reply:
Agreed
I created a cmd runner for testing. for some reason the cmd/ directory isn't getting ignored in .gitignore.